### PR TITLE
Problem: autosync don't work correctly when status reply is not delivered

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: change wallet when sync is 100% , also add timeout for get status

### CODE REVIEW

File: client-core/src/synchronizer/auto_sync_core.rs [hunk 1]
```diff
@@ -447,7 +449,15 @@ where
...
+
+        if self.get_current_height() < self.max_height {
+            self.send_request_block()
+        } else {
+            log::info!("polling_get_blocks, sync is complete, go to next wallet");
+            self.change_to_wait();
+            self.change_wallet();
+            Ok(())
+        }
     }
```
Comment (leejw51crypto) [inline client-core/src/synchronizer/auto_sync_core.rs]: feat by @lezzokafka

**Comment by calvinaco** [comment]
Can you explain a bit more about this PR?

**Comment by leejw51crypto** [comment]
yes, in autosync,
we found that in some situation
in GetBlocks state,
wallet doesn't change.

so i added in polling, if sync is complete,
i activated changing wallet.

**Comment by leejw51crypto** [comment]
sure, i'll try to unit test finite states.

**Comment by tomtau** [comment]
bors try

**Comment by tomtau** [comment]
@calvinaco any insight why this one failed: https://travis-ci.org/crypto-com/chain/jobs/589845794#L5849 ?

**Comment by calvinaco** [comment]
The integration tests already passed. So it is `docker-compose logs -t --tail="all"` causing problem. My suspicious is the logs is too big but I need to retry it one more time to verify first.

I will wait for bors to do its job first.

**Pull request closed without merging**